### PR TITLE
docs: cost comparisons for beta and polygon

### PR DESCRIPTION
**Pull request by superical**
## Summary
A page for showing the comparison in costs between the current version and beta, and also between Ethereum and Polygon.

- To eliminate (or at least reduce) the on-going concern of costs and gas fee complains
- To encourage people to start trying out the beta version
- For IMDA to use as supporting documentation for purchasing ethers as discussed in one of the sprints

## Issues
* https://www.pivotaltracker.com/story/show/182022232

### CODE REVIEW

File: website/docs/docs-section/appendix/savings-comparisons.mdx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+title: Costs & Savings (Beta)
+sidebar_label: Costs & Savings (Beta)
+---
+
+import { FiatLabel } from "../../../src/components/FiatLabel";
+import { BoxTag } from "../../../src/components/BoxTag";
+import { useFetchGasCost } from "../../../src/hooks";
+
+Are you a new user of TradeTrust or looking at deploying a new Token Registry (Transferable Documents)? If you are, we'd totally suggest that you try out our new V4 beta version!
+
```
Comment (simboonlong) [inline website/docs/docs-section/appendix/savings-comparisons.mdx:12]: sorry got abit confused on versions, what's in v4 and what to look out for? we now have v3 and v4 in betas? 🤔

Comment (simboonlong) [reply website/docs/docs-section/appendix/savings-comparisons.mdx:12]: ah ok so there's `OpenAttestation V3 (Beta)` and `Token Registry V4 (Beta)`

File: website/docs/docs-section/appendix/savings-comparisons.mdx [hunk 2]
```diff
@@ -0,0 +1,186 @@
+---
+id: savings-comparisons
+title: Costs & Savings (Beta)
+sidebar_label: Costs & Savings (Beta)
```
Comment (simboonlong) [inline website/docs/docs-section/appendix/savings-comparisons.mdx:4]: nitpick:

```suggestion
title: Token Registry V4 (Beta)
sidebar_label: token_registry V4 (Beta)
```

Comment (superical) [reply website/docs/docs-section/appendix/savings-comparisons.mdx:4]: Should the `sidebar_label` be the same as the `title`? 'Cos I think the `sidebar_label` will be displayed on the side, right?

> sidebar_label: The text shown in the document sidebar and in the next/previous button for this document. If this field is not present, the document's sidebar_label will default to its title.

Apparently, if they are the same, we can even omit the `sidebar_label`. Didn't know about this. haha

Comment (superical) [reply website/docs/docs-section/appendix/savings-comparisons.mdx]: Updated the page title in https://github.com/Open-Attestation/documentation-website/pull/128/commits/2e34d6cf63a4055937cab6e562c8254aea2e23b3

**Review by simboonlong — APPROVED** [review]

**Pull request merged**